test/e2e/storage/volumes.go

```diff
@@ -503,24 +503,21 @@ var _ = utils.SIGDescribe("Volumes", func() {
 	Describe("vsphere [Feature:Volumes]", func() {
 		It("should be mountable", func() {
 			framework.SkipUnlessProviderIs("vsphere")
+			vspheretest.Bootstrap(f)
+			nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
```

Comment:
Why is this call needed when Bootstrap sets the TestContext to the nodeList?

Reply:
With `Bootstrap(f)` call we are collecting `nodeinfo` objects for all nodes including `master` and other `not schedulable` nodes.

but in the test we need nodeinfo object for scheduleable and ready node, so that pod can be scheduled on that node. You can refer `vsphere_volume_diskformat.go` to understand how we are setting node labels and schedule pod on specific node.